app/utils/general.py

```diff
@@ -72,11 +73,59 @@ def log_error(context: str) -> str:
     return message
 
 
-def seq_id() -> str:
+class Timestamp:
```

Comment:
Originally, I was going to deduce the age of the posted item list from the list key, so I added functionality to decode the seq_ids to get their component parts (hence the class).  But later when I switched to a sorted set of item lists (keeping the "ready time" in the sort key) this change became unnecessary.  I could have reverted it but it seems harmless because the base timestamp logic is the same as the seq_id logic.